salt/base/salt.sls

```diff
@@ -80,21 +80,50 @@ salt-master:
     - require:
       - pkg: consul-pkgs
 
-/srv/psf_known_hosts:
+/srv/public:
+  file.directory:
+    - user: root
+    - group: root
+    - mode: "0644"
```

Comment:
we need "0755" here. Directory permissions 0644 -> `rw-r--r--` won't allow anyone to change directory in without execute, so 0755 -> `rwxr-xr-x` will let nginx read from the directory via try_files